### ISSUE TITLE
[Homematic] no update of ccu variables received

### ISSUE THREAD

**Issue by ptweety** [openhab/openhab2-addons#1663]
Hi,
I'm using openhab2 beta5 together with a ccu2 (firmware 2.25.15) and the homematic binding. The bridge has been discovered automatically via paperui. Channels and items have been created automatically via simple mode.

I've a rule (C) in the ccu to update a system variable (homematic_GATEWAY_EXTRAS_CCU_ccu_GWE00000000_1_User1_Presence) on button press (homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT)

Furthermore I've a rule (O) in openhab to check for the current state of the item linked to the system variable. If this item needs an update I send a command to the ccu (homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT.sendCommand(ON)) 

Both rules are working as expected. In ccu I can see the update of the system variable, but openhab does not receive this update. Therefore openhab and ccu are out of sync and the rule (O) tries to fire the same command over and over again.

```
Group:Switch:OR(ON, OFF)	gPresence_P1		"Me"			<boy_3>		(gPresence)
Switch				Presence_P1_Override	"... ist doch da"				(gPresence_P1)
Switch				Presence_P1_PhoneWL	"... per WLAN [%s]"		<network>	(gPresence_P1)	{fritzboxtr064="maconline:xx-xx-xx-xx-xx-xx" }
```

```
rule "gPresence_P1 received update"
when
	Item gPresence_P1 received update or
	Time cron "0 0/5 * 1/1 * ? *"
then {
	if ( homematic_GATEWAY_EXTRAS_CCU_ccu_GWE00000000_1_User1_Presence.state != gPresence_P1.state ) {
		homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT.sendCommand(ON)
		logInfo("RULE", homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT.name + ": send command ON")
	}
}
end
```

```
10:35:48.191 [INFO ] [.eclipse.smarthome.model.script.RULE] - homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT: send command ON
10:35:48.207 [INFO ] [smarthome.event.ItemCommandEvent    ] - Item 'homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT' received command ON
10:35:48.215 [INFO ] [marthome.event.ItemStateChangedEvent] - homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT changed from OFF to ON
10:35:50.247 [INFO ] [marthome.event.ItemStateChangedEvent] - homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT changed from ON to OFF
...
10:36:51.625 [INFO ] [.eclipse.smarthome.model.script.RULE] - homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT: send command ON
10:36:51.631 [INFO ] [smarthome.event.ItemCommandEvent    ] - Item 'homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT' received command ON
10:36:51.634 [INFO ] [marthome.event.ItemStateChangedEvent] - homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT changed from OFF to ON
10:36:53.672 [INFO ] [marthome.event.ItemStateChangedEvent] - homematic_HM_RCV_50_ccu_BidCoS_RF_1_PRESS_SHORT changed from ON to OFF
```

**Comment by ptweety** [comment]
Hi, I did my test again with log level TRACE. Maybe this helps:

[trace.log.zip](https://github.com/openhab/openhab2-addons/files/702527/trace.log.zip)

**Comment by MHerbst** [comment]
@ptweety  Is this problem solved with the answer from #1794? Then please set this issue to solved.
